libbeat/conditions/matcher.go

```diff
@@ -86,7 +86,7 @@ func (c Matcher) Check(event ValuesMap) bool {
 				return false
 			}
 
-		case []string:
+		case []interface{}:
```

Comment:
What events were you seeing where it had `[]interface{}` rather than `[]string`? I assume this code would need to handle both data types. My guess is that `[]interface{}` is found in data parsed by Filebeat and that `[]string` is found in events that are natively generated in a Beat (such as a list of `tags`).

Reply:
You're completely right. Thanks. I got confused with maps from decoders where arrays are always of type []interface{}.